tests/unit/helpers/events.test.js

```diff
@@ -769,6 +770,26 @@ describe('removeEventsByContractStatus', () => {
 
     await EventHelper.removeEventsByContractStatus(contract);
     sinon.assert.called(EventAggregateStub);
-    sinon.assert.calledWith(EventDeleteManyStub, { startDate: { $gt: contract.endDate }, subscription: aggregation[1].sub._id, isBilled: false });
+    sinon.assert.calledWith(EventDeleteManyStub, { startDate: { $gt: contract.endDate }, subscription: { $in: [aggregation[1].sub._id] }, isBilled: false });
+  });
+
+  it('should return a 400 error if no contract provided', async () => {
+    try {
+      await EventHelper.removeEventsByContractStatus();
+    } catch (e) {
+      expect(e).toEqual(Boom.badRequest());
+    }
+  });
+
+  it('should return a 404 error if no subscriptions match contract status', async () => {
+    try {
+      const contract = { status: CUSTOMER_CONTRACT, endDate: moment().toDate(), user: userId, customer: customerId };
```

Comment:
Plutôt `moment().toDate()` ou `Date.now()` ?

Reply:
C'est un dilemme cornélien ! :D Mais sachant qu'on importe ce superbe outil qu'est 'moment'... `new Date()` est trop old school ^^

Reply:
Tu es trop `in` !
Mais ça me va dans tous les cas.